libtcmu_log.c

```diff
@@ -411,10 +412,15 @@ static int output_to_fd(int pri, const char *timestamp,
 	if (pid <= 0)
 		return -1;
 
+	if (tcmu_get_thread_name(&pname))
```

Comment:
We can probably just drop the function and make this a lot more simple for now. Just do a:

char pname[TCMU_THREAD_NAME_LEN];
...
pthread_getname_np(pthread_self(), pname, TCMU_THREAD_NAME_LEN);

in this function.

Reply:
sure.